jpype/_core.py

```diff
@@ -95,48 +95,84 @@ def _hasClassPath(args):
 
 _JVM_started = False
 
-def startJVM(jvm=None, *args, **kwargs):
+def startJVM(*args, **kwargs):
```

Comment:
wildcard arguments are a pain! Please keep jvm (also for backward compatibility reasons).

Reply:
I believe it does the same thing here.

I wanted to be able to support 
```
  # old
   jpype.startJVM(getDefaultJVMPath(), "-ea", '-Djava.class.path=%s'%getClassPath())
  # Better
   jpype.startJVM(getDefaultJVMPath(), '-ea', classpath = getClassPath())
   jpype.startJVM(getDefaultJVMPath(), '-ea')
  #  Simple
   jpype.startJVM('-ea')
 ```

Reply:
Just to be clear ``jvm`` as the first argument before the ``*args` meant you had to specify the path even if you just wanted to start a jvm with default.  Passing in None there is the alternative, but looked really ugly to me.  I can change it back.  

Also minor issue.... i think the argument should be ``jvmpath`` rather than ``jvm``.  I want to reserve ``jvm`` for when we actually can start more than one machine.